packages/@webex/internal-plugin-metrics/test/unit/spec/call-diagnostic/call-diagnostic-metrics.ts

```diff
@@ -2367,5 +2368,20 @@ describe('internal-plugin-metrics', () => {
         assert.isFalse(cd.isServiceErrorExpected(999999));
       });
     });
+
+    describe.only('#setDeviceInfo', () => {
```

Comment:
I guess this is my mistake you added `describe.only`, would you remove this ??
@mkesavan13

Reply:
Sorry about that. Removed only